jenkinsfiles/Jenkinsfile.update-journal-pr

```diff
@@ -3,7 +3,13 @@ tag = params.tag
 
 elifePipeline {
     stage "Launch stack", {
```

Comment:
Tiny potential change could be to update the stage name to 'Launch stack if not running' or something.

Reply:
if Jenkins gets different column names when running a pipeline it messes with the job history. A better option would be to 'skip' the stage, making it grey instead of green, if that's possible. I'll take a look.

Reply:
ok, from what I can see the change I've made *should* work, the `import` statement *should* be [whitelisted in the Groovy sandbox now](https://issues.jenkins.io/browse/JENKINS-54322?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel). If it doesn't work I'll do some extra debugging